__init__.py

```diff
@@ -129,8 +131,28 @@ def _b64(b):
         domain_key_file.write(domain_key_contents)
 
       log.info("Generate csr...")
-      proc = subprocess.Popen(["openssl", "req", "-new", "-sha256", "-key", domain_key, "-subj", "/CN=" + domain],
-                              stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+      if len(domains) > 1:
+        domain_config_file = tempfile.NamedTemporaryFile()
+        default_conf = open("/etc/ssl/openssl.cnf").readlines()
```

Comment:
Alas, I really do not know what is needed or not in the conf file.
The best I can do is to inline a copy of the openssl.cnf file in the script, but I don't know if we want it or not.

Also, on Windows the NamedTemportaryFile should not be openned by the openssl command without being closed before in the script (and thus, created with the `delete = False` parameter, and being deleted manually after the run of openssl.